### PR TITLE
Added two new arguments to simplify command line call

### PR DESCRIPTION
**Pull request by JAlvarezJarreta**
To avoid problems with extremely large command lines when the number of fasta files is really high (above 10k files), I have added two new arguments (named `-A` and `-B` after their existing counterparts) to pass the large list of fasta files inside a text file (`-A`) and the newick tree string inside another file (`-B`).

I have also added a explicit call to Python 2 in the Makefile because the existing code doesn't compile in Python 3 (may need to be updated with the upcoming Python 2.7 EOL).

### CODE REVIEW

File: OrtheusC.c [hunk 1]
```diff
@@ -8,6 +8,9 @@
 #include <time.h>
 #include <assert.h>
 #include <ctype.h>
+#include <dirent.h>
```
Comment (muffato) [inline OrtheusC.c]: Not needed any more

File: OrtheusC.c [hunk 2]
```diff
@@ -332,6 +335,47 @@ struct List *parseSequences(char **argv, int64_t inputSequenceNumber, int64_t *s
     return seqs;
 }
 
+struct List *parseSequencesFromFile(char *inputSequencesFile, int64_t *seqNo, int64_t **seqLengthsA) {
```
Comment (muffato) [inline OrtheusC.c]: Add comments here and in the previous function to explain both have to be kept in sync

File: OrtheusC.c [hunk 3]
```diff
@@ -659,7 +732,11 @@ int main(int argc, char *argv[]) {
     }
     st_logInfo("Newick-Tree seems okay\n");
     //parse sequences
-    seqs = parseSequences(inputSequences, inputSequenceNumber, &seqNo, &seqLengths);
+    if (inputSequenceNumber < INT_MAX) {
+        seqs = parseSequences(inputSequences, inputSequenceNumber, &seqNo, &seqLengths);
+    } else {
```
Comment (muffato) [inline OrtheusC.c:712]: Add `if (inputSequencesFile)` to avoid entering the function with a _NULL_

File: OrtheusC.c [hunk 4]
```diff
@@ -659,7 +732,11 @@ int main(int argc, char *argv[]) {
...
+        seqs = parseSequencesFromFile(inputSequencesFile, &seqNo, &seqLengths);
+    }
```
Comment (muffato) [inline OrtheusC.c:715]: Add an `else` to complain about no input sequences given ?

**Review by muffato — CHANGES_REQUESTED** [review]
Some relatively minor changes. The code is otherwise clean and clear

File: OrtheusC.c (hunk 2 above)
Comment (JAlvarezJarreta) [reply OrtheusC.c]: Although not the most efficient solution, I have managed to merge these two methods into one.

**Review by muffato — APPROVED** [review]

**Pull request merged**